scenarios/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10
```

Comment:
Should we use Python 3.12 instead, since it is what the latest release is based on?

Reply:
This is because the library acapy-minimal-example library targets 3.10. This is an issue with using an external library for this. I believe it would be pretty straightforward to get a PR for the indicio library to upgrade to 3.12 and then we could upgrade it here.

We are also targeting the `main` branch of the indicio github. Targeting a release tag would be a better strategy.